.circleci/config.yml

```diff
@@ -195,6 +195,15 @@ jobs:
           test_params: VALGRIND=1
           getredis_params: --valgrind
 
+  tls:
+    docker:
+      - image: redisfab/rmbuilder:6.2.5-x64-bullseye
```

Comment:
Might as well use 6.2.6 if we're already here.

Reply:
Fixed